packages/preview/touying-buaa/0.2.0/lib.typ

```diff
@@ -0,0 +1,30 @@
+// Stargazer theme - adapted for BUAA
+#import "@preview/touying:0.5.0": *
```

Comment:
#import "@preview/touying:0.5.2": *

Reply:
I saw https://github.com/touying-typ/touying/commit/334f8a2190ae1767c269b75474c7d33f0aa8f0bc . Would it indicate releasing 0.5.3 soon?

Reply:
I don't think so, because it's a minor bug and it may be a week or two, or even longer, before 0.5.3 arrives. Maybe when typst 0.12.0 comes out, the incompatible content will be dealt with as well.

Reply:
Ok, let us use 0.5.2 for now.